### PR TITLE
Update adoptium api to latest builds

### LINKED ISSUE #2909: Switch to the Adoptium api to get the latest nightly builds
Currently we are still using the Adoptopenjdk api to get expected jdk, which should switch to the Adoptium api to get the latest nightly builds.

Update

`https://api.adoptopenjdk.net/v3/binary/latest/${JDK_VERSION}/${release_type}/${os}/${arch}/jdk/${JDK_IMPL}/${heap_size}/adoptopenjdk`

to 
`https://api.adoptium.net/v3/binary/latest/${JDK_VERSION}/${release_type}/${os}/${arch}/jdk/${JDK_IMPL}/${heap_size}/adoptium?project=jdk`

 https://github.com/adoptium/aqa-tests/blob/master/get.sh#L229

### PR DESCRIPTION
**Pull request by jyami-kim**
Fixes #2909 

I fixed this issue at GHC. it is my first open source contribution :)
if you have any comment, please good review.

### CODE REVIEW

**Review by sophia-guo — APPROVED** [review]
LGTM. Thanks @jyami-kim

**Comment by sophia-guo** [comment]
Grinder failed: 
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1531/console. Need to see what's the issue.

**Comment by sophia-guo** [comment]
Looks like servers down. With original api got the same error. 

```
The webpage at https://api.adoptopenjdk.net/v3/binary/latest/11/ea/linux/x64/jdk/hotspot/normal/adoptopenjdk might be temporarily down or it may have moved permanently to a new web address.
```

**Comment by jyami-kim** [comment]
I think, its release_type ea is not support that version (or temporarily down).
in this swagger https://api.adoptium.net/q/swagger-ui/#/Binary/getBinary
When I try to release type parameter "ga", error code is not 400. (when I try to ea, error code is 400.)

**Comment by smlambert** [comment]
This PR looks good @jyami-kim ! it has interestingly found a possible problem with the API.  :) 

We have someone investigating the issue with the API and will merge this once it is resolved.

File: get.sh [hunk 1]
```diff
@@ -226,7 +226,7 @@ getBinaryOpenjdk()
 		if [ "$SDK_RESOURCE" == "releases" ]; then
 			release_type="ga"
 		fi
-		download_url="https://api.adoptopenjdk.net/v3/binary/latest/${JDK_VERSION}/${release_type}/${os}/${arch}/jdk/${JDK_IMPL}/${heap_size}/adoptopenjdk"
+		download_url="https://api.adoptium.net/v3/binary/latest/${JDK_VERSION}/${release_type}/${os}/${arch}/jdk/${JDK_IMPL}/${heap_size}/adoptium?project=jdk"
 		info_url="https://api.adoptopenjdk.net/v3/assets/feature_releases/${JDK_VERSION}/${release_type}?architecture=${arch}&heap_size=${heap_size}&image_type=jdk&jvm_impl=${JDK_IMPL}&os=${os}&project=jdk&vendor=adoptopenjdk"
```
Comment (llxia) [inline get.sh:231]: Could you also update info_url? 
- change `api.adoptopenjdk.net` to `api.adoptium.net`
- I think `vendor=adoptopenjdk` should be `vendor=eclipse`

**Comment by sophia-guo** [comment]
We might not need to wait the fix. As the issue also affects current adoptopenjdk api query with that staticlibs enabled.

**Review by llxia — APPROVED** [review]
LGTM

**Comment by sophia-guo** [comment]
Believe this has been fixed by https://github.com/adoptium/ci-jenkins-pipelines/pull/194

reran: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1586/console

**Pull request merged**